.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -12,11 +12,22 @@ body:
     validations:
       required: true
 
+  - type: input
+    id: renovate-version-if-selfhosting
+    attributes:
+      label: If you're self-hosting Renovate, tell us what version of Renovate you run.
+      description: |
+        The Renovate maintainers only support the latest version of Renovate.
+        The Renovate team will only create bugfixes for an older version if the hosted app needs to stay on an older major version for a short time or if some critical bug needs to be fixed and the new major is blocked.
+        If you're using the hosted app, you don't need to do anything, as the Renovate maintainers update the hosted app regularly.
+        If you're self hosting Renovate, use the latest release if possible.
```

Comment:
Source for the `description`: https://docs.renovatebot.com/faq/#which-renovate-versions-are-officially-supported